orchestrator/careplancontributor/service.go

```diff
@@ -128,6 +128,12 @@ func (s Service) RegisterHandlers(mux *http.ServeMux) {
 		log.Info().Msgf("Redirecting to %s", s.frontendUrl)
 		http.Redirect(response, request, s.frontendUrl, http.StatusFound)
```

Comment:
Don't redirect to the frontend URL, because the it shows the frontend enrolment UI without any data (or even "no user session", or something). Either;
- Redirect to URL in `Referer` header, or
- Render some text that says "you have been logged out" if there's no Referer header.